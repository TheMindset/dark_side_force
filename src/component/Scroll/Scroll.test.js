import React from 'react'
import { shallow } from 'enzyme'
import Scroll from './Scroll'


describe('Scroll', () => {
  const mockSelectedMovie = {
    title: 'The GodFather',
    episode: 2,
    scrollText: 'HÃ© bambino...'
  }

  let wrapper = shallow (<Scroll selectedMovie={mockSelectedMovie}/>)

  test('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
