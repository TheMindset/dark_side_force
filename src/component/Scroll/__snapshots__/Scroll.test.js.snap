// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Scroll should match the snapshot 1`] = `
<div
  className="scroll"
>
  <div
    className="fade"
  >
    <section
      className="star-wars"
    >
      <div
        className="crawl"
      >
        <div
          className="title"
        >
          <p>
            Episode 
            2
          </p>
          <h1>
            The GodFather
          </h1>
        </div>
        <p>
          HÃ© bambino...
        </p>
      </div>
    </section>
  </div>
</div>
`;
